datafusion/physical-expr/src/aggregate/average.rs

```diff
@@ -154,6 +159,14 @@ impl Accumulator for AvgAccumulator {
         Ok(())
     }
 
+    fn retract_batch(&mut self, values: &[ArrayRef]) -> Result<()> {
+        let values = &values[0];
+        self.count -= (values.len() - values.data().null_count()) as u64;
+        let delta = sum_batch(values, &self.sum.get_datatype())?;
+        self.sum = self.sum.sub(&delta)?;
```

Comment:
I like giving a name to the change (i.e. `delta`). I suggest doing the same in `update_batch`.